community/rds/aurora-serverless/template.yml

```diff
@@ -0,0 +1,70 @@
+---
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: Aurora serverless cluster
+
+Parameters:
+  DatabaseName:
+    Type: String
+  MasterUserPassword:
+    Type: AWS::SSM::Parameter::Value<String>
+    Description: Parameter Store name
+  VpcId:
+    Type: String
+  VpcSecurityGroupId:
+    Type: String
+  BastionKeyName:
+    Type: String
+    Description: EC2 key used to connect to the bastion host
+    Default: Default
+
+Resources:
+  Cluster:
+    Type: AWS::RDS::DBCluster
+    Properties:
+      Engine: aurora
+      EngineMode: serverless
+      EngineVersion: 5.6
+      DatabaseName: !Ref DatabaseName
+      MasterUsername: root
+      MasterUserPassword: !Ref MasterUserPassword
+      DBClusterIdentifier: !Ref AWS::StackName
+      BackupRetentionPeriod: 35
+      DeletionProtection: true
+      VpcSecurityGroupIds:
+        - !Ref VpcSecurityGroupId
+
+  SecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties: 
+      GroupDescription: !Sub 'Bastion for ${AWS::StackName}'
+      SecurityGroupEgress:
+        - DestinationSecurityGroupId: !Ref VpcSecurityGroupId
+          IpProtocol: tcp
+          FromPort: 3306
+          ToPort: 3306
+      SecurityGroupIngress: []
+      VpcId: !Ref VpcId
+
+  Bastion:
+    Type: AWS::EC2::Instance
+    Properties:
+      DisableApiTermination: true
+      ImageId: ami-035be7bafff33b6b6
```

Comment:
Hard coding AMI IDs is never a good idea, plus I'm having trouble finding this particular AMI.  An easy trick is to use the managed parameter store values to grab the latest Amazon Linux AMI dynamically:
https://aws.amazon.com/blogs/compute/query-for-the-latest-amazon-linux-ami-ids-using-aws-systems-manager-parameter-store/